feat(underscore-redirects): add `base` to input paths (#8953)
withastro/astro: .changeset/odd-maps-glow.md

---

> '@astrojs/underscore-redirects': minor

Reviewer: Since this package is v0, should this be a patch instead? Seems like it should be non-breaking.

---

> Adds the base path in front for input paths

Reviewer 2: ```suggestion
Adds the base path as prefix for input paths
```
I like the way you said it in the PR description better.  😄  I think "prefix" holds more meaning in context.
